.github/workflows/release.yml

```
@@ -473,6 +492,104 @@ jobs:
+            }' > latest.json
+
+          cat latest.json
+
+      - name: Upload latest.json to GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: v${{ steps.info.outputs.version }}
+          name: ${{ inputs.release_type == 'hotfix' && steps.info.outputs.tag_name || format('v{0}', steps.info.outputs.version) }}
+          body_path: releasenote.md
+          prerelease: ${{ steps.info.outputs.prerelease }}
+          overwrite: true
```

actionlint (1.7.12)

[error] input "overwrite" is not defined in action "softprops/action-gh-release@v2". available inputs are "append_body", "body", "body_path", "discussion_category_name", "draft", "fail_on_unmatched_files", "files", "generate_release_notes", "make_latest", "name", "overwrite_files", "prerelease", "preserve_order", "previous_tag", "repository", "tag_name", "target_commitish", "token", "working_directory"

(action)